rosetta/transactions/sign_transaction.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package transactions
+
+import (
+	"encoding/hex"
+
+	"github.com/onflow/flow-go-sdk"
+
+	"github.com/optakt/flow-dps/rosetta/failure"
+	"github.com/optakt/flow-dps/rosetta/object"
+)
+
+// SignTransaction adds the given signature to the transaction.
+func (p *Parser) SignTransaction(tx *flow.Transaction, signature object.Signature) error {
```

Comment:
We should still have a return here. Using input parameters as output is not good practice in Go, even when using pointers. Best to create a copy, actually.